spinn_front_end_common/utilities/report_functions/bit_field_compressor_report.py

```diff
@@ -56,11 +58,15 @@ def bitfield_compressor_report():
     file_name = os.path.join(FecDataView.get_run_dir_path(), _FILE_NAME)
     try:
         with open(file_name, "w", encoding="utf-8") as f:
-            return _write_report(f)
+            _write_report(f)
     except IOError:
         logger.exception("Generate_placement_reports: Can't open file"
                          " {} for writing.", _FILE_NAME)
-        return None
+        return
+    except NoProvenanceDatabaseException:
+        logger.exception(
+            "No proveance found to write bitfield_compressor_report")
+        return
```

Comment:
Probably not needed here either?

Reply:
NoProvenanceDatabaseException is never raised so whole Exception classed nuked.